contributors/design-proposals/standardization-of-development-process.md

```diff
@@ -0,0 +1,282 @@
+responsible SIG or working group. In order to help combat the explosion of GitHub
+issues against `kubernetes/kubernetes` it is further proposed for each chartered
+SIG to create a top level repository within the `kubernetes` GitHub organization
+where implementation issues would be filed and linked to in the RFC.
+
+### RFC Editor Role
+
+Taking a cue from the [Python PEP process][], I believe that a group of RFC editors
+will be required to make this process successful; the job of an RFC editor is
+likely very similar to the [PEP editor responsibilities][] and will hopefully
+provide another opportunity for people who do not write code daily to contribute
+to Kubernetes.
+
+[Python PEP process]: https://www.python.org/dev/peps/pep-0001/
+[PEP editor responsibilities]: https://www.python.org/dev/peps/pep-0001/#pep-editor-responsibilities-workflow
+
+### Important Metrics
+
+I believe the primary metrics which would signal the success or failure of the RFC
+process would be
+
+- distribution of RFC merge times
+- RFC rejection rate
+- distribution of time between RFC acceptance and implementation issue creation
+- PRs referencing an RFC merged per week
+
+### Customization of the RFC Process
+
+Given the general independence of SIGs I believe it will be important for SIGs to
+decide if they would like to opt out of the RFC process entirely or if they would
+like to modify the RFC process to better serve their needs. For example I can
+imagine RFCs targeting SIG Architecture or SIG API Machinery would likely require
+a section on backwards compatibility and migration strategies. I believe the best
+place for a SIG to describe its development process would be in its charter.
+
+### Prior Art
+
+The RFC process as proposed was essentially stolen from the [Rust RFC process] which
+itself seems to be very similar to the [Python PEP process][]
+
+[Rust RFC process]: https://github.com/rust-lang/rfcs
+
+## Graduation Criteria
+
+Before we can consider this process successful for our current size I believe we
+should hit at least the following milestones
+
+- a release note draft can be generated by referring to RFC content
+- a road map can be generated by referring to RFC content
+
+The process for creating draft release notes and development status reports
+should eventually rely largely on automation, but I do not believe that the
+implementation of such automation should prevent the RFC process from being
+considered stable. I can certainly imagine a future where a release notes draft
+is produced nightly along with the collection of binaries that users deploy.
+
+## Drawbacks
+
+Any additional process has the potential to engender resentment within the
+community. There is also a risk that the RFC process as designed will not
+sufficiently address the scaling challenges we face today. PR review bandwidth is
+already at a premium and we may find that the RFC process introduces an unreasonable
+bottleneck on our development velocity.
+
```

Comment:
I think we would pick a day. Perhaps SIGs could go through what's currently in `design-proposals` and prune out non implemented designs but I think we'd probably archive the old design documentation that had no active work around its implementation.